<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试窗口_1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_text1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入关键词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
